openrr-base/src/utils.rs

```diff
@@ -0,0 +1,38 @@
+use arci::BaseVelocity;
+
+#[derive(Debug, Clone, Copy)]
+pub struct BaseVelocityTimestamped {
+    pub base_velocity: BaseVelocity,
+    pub timestamp: std::time::Instant,
+}
+
+impl Default for BaseVelocityTimestamped {
+    fn default() -> Self {
+        Self {
+            base_velocity: BaseVelocity::default(),
+            timestamp: std::time::Instant::now(),
+        }
+    }
+}
```

Comment:
I think it's nice to add a function to create this with velocity and Instant::now();

```rust
fn new_with_now(vel :BaseVelocity) -> Self {
}
```